AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContextDelegateWrapper.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAuthTokenContextDelegate.h"
+#import "MSAuthTokenDelegate.h"
+
+/**
+ * MSAuthTokenContext:addDelegate takes a MSAuthTokenContextDelegate, but our new MSAuthTokenDelegate has a different form, 
+ * so we need to create a wrapper which conforms to MSAuthTokenContextDelegate.
+ */
+@interface MSAuthTokenContextDelegateWrapper : NSObject <MSAuthTokenContextDelegate>
+
+@property id<MSAuthTokenDelegate> delegate;
+
+@property MSAuthTokenCompletionHandler authTokenCompletionHandler;
+
+/**
+ * Create a delegate wrapper which will call the delegates for both MSAuthTokenContextDelegate and MSAuthTokenDelegate.
```

Comment:
I wouldn't block on this, but mind tense when writing documentation - in other methods there are sentences like "initializes" rather than "initialize." So maybe this could be "Creates"